rosa_backing_up_and_restoring_applications/backing-up-applications.adoc

```diff
@@ -15,28 +15,39 @@ This is a two stage process:
 
 include::modules/oadp-preparing-aws-credentials.adoc[leveloffset=+1]
 
-include::modules/oadp-installing-oadp-rosa-sts.adoc[leveloffset=+1]
+include::modules/installing-oadp-rosa-sts.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 [id="oadp-addtl-resources_{context}"]
 .Additional resources
 
 * xref:../rosa_backing_up_and_restoring_applications/backing-up-applications.adoc#oadp-preparing-aws-credentials_rosa-backing-up-applications[Preparing AWS credentials]
 
+//Added the following two modules to make a more complete guide.
+
+[id="oadp-rosa-backing-up-and-cleaning-example"]
+== Backing up workloads on OADP with ROSA STS
+
+include::modules/performing-a-backup-oadp-rosa-sts.adoc[leveloffset=+2]
+
+include::modules/cleanup-a-backup-oadp-rosa-sts.adoc[leveloffset=+2]
+
 [id="rosa-backing-up-applications-known-issues"]
 == Known issues
-.Restic, Kopia, and DataMover are not supported or recommended
 
-* link:https://issues.redhat.com/browse/OADP-1054[CloudStorage: openshift-adp-controller-manager crashloop seg fault with Restic enabled]
-* (Affects OADP 1.1.x_ only): link:https://issues.redhat.com/browse/OADP-1055[CloudStorage: bucket is removed on CS CR delete, although it doesn't have "oadp.openshift.io/cloudstorage-delete": "true"]
+* Restic, Kopia, and DataMover are not supported or recommended.
+* link:https://issues.redhat.com/browse/OADP-1054[CloudStorage: openshift-adp-controller-manager crashloop seg fault with Restic enabled].
+* (Affects OADP 1.1.x_ only): link:https://issues.redhat.com/browse/OADP-1055[CloudStorage: bucket is removed on CS CR delete, although it doesn't have "oadp.openshift.io/cloudstorage-delete": "true"].
 
 [role="_additional-resources"]
 [id="additional-resources_rosa-backing-up-applications"]
 == Additional resources
 
-* link:https://docs.openshift.com/rosa/rosa_architecture/rosa-understanding.html[Understanding ROSA with STS]
-* link:https://docs.openshift.com/rosa/rosa_getting_started/rosa-sts-getting-started-workflow.html[Getting started with ROSA STS]
-* link:https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.html[Creating a ROSA cluster with STS]
-* link:https://docs.openshift.com/container-platform/4.13/backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.html[About installing OADP]
-* link:https://docs.openshift.com/container-platform/4.13/storage/container_storage_interface/persistent-storage-csi.html[Configuring CSI volumes]
-* link:https://docs.openshift.com/rosa/rosa_architecture/rosa_policy_service_definition/rosa-service-definition.html#rosa-sdpolicy-storage_rosa-service-definition[ROSA storage options]
+* For information about ROSA architecture, see xref:../rosa_architecture/rosa-understanding.adoc#rosa-understanding-about_rosa-understanding[Understanding ROSA].
```

Comment:
If you feel that the "For information..." lead-in sentence is important, keep them. If not, you can convert all links to the following format:

* xref:../rosa_planning/rosa-sts-aws-prereqs.adoc#rosa-sts-aws-prereqs[AWS prerequisites for ROSA with STS]
Examples are here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections

Reply:
This is not one of my topics. I'm not sure how this got pulled into this PR? :face_with_diagonal_mouth: